src/odb/src/db/dbITerm.cpp

```diff
@@ -367,11 +367,19 @@ void dbITerm::connect(dbNet* net_)
   _dbNet* net = (_dbNet*) net_;
   _dbBlock* block = (_dbBlock*) iterm->getOwner();
 
+  _dbInst* inst = iterm->getInst();
+  if (!net_) {
+    inst->getLogger()->error(utl::ODB,
+                             440,
+                             "Attempt to connect iterm {}/{} to a null net",
+                             inst->_name,
```

Comment:
Shouldn't this use the getName()?

Reply:
It will give the same result but I could replace it if you think it is better.

Reply:
I'll change it as it avoids a bit of redundancy

Reply:
I think if we have the function it makes sense to use it.